inc/scripts.php

```diff
@@ -5,6 +5,20 @@
  * @package _s
  */
 
+/**
+ * Enqueue Tailwind.
+ *
+ * Load Tailwind dependencies in editor and frontend.
+ *
+ * @author WDS
+ */
+function _s_tailwind_scripts() {
+	// Register styles & scripts.
+	wp_enqueue_style( 'tailwind', get_stylesheet_directory_uri() . '/build/tailwind.css', array(), $asset_file['version'] );
```

Comment:
Getting these on the frontend when I load it up:

`Notice: Undefined variable: asset_file in /Applications/MAMP/htdocs/wd_s/wp-content/themes/wd_s/inc/scripts.php on line 17`
`Notice: Trying to access array offset on value of type null in /Applications/MAMP/htdocs/wd_s/wp-content/themes/wd_s/inc/scripts.php on line 17`

Reply:
It looks like my `wp debug` wasn't on. I got this resolved.